src/routes/index.jsx

```diff
@@ -1,16 +1,32 @@
+
 import React from 'react';
 import { BrowserRouter, Route } from 'react-router-dom';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faStar, faHeart, faReply } from '@fortawesome/free-solid-svg-icons';
 import Header from '../components/Header';
 import Footer from '../components/Footer';
+// eslint-disable-next-line import/no-named-as-default
 import HomeView from '../containers/HomeView';
 import UserProfile from '../components/user/UserProfile';
+import Login from '../components/Login';
+import RegisterForm from '../components/RegisterForm';
+import GetAnArticle from '../containers/GetAnArticle';
```

Comment:
@Etomovich, why are you importing Login, RegisterForm and GetAnArticle here?

Reply:
Register form is another person import but GetAnArticle is the container that holds both fetch an article and its comments. then it is registered under the route `/article/:slug`